pkgs/games/sgfutils/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, openssl
+, iconv
+, makeWrapper
+, imagemagick
+, makeFontsConf
+}:
+stdenv.mkDerivation
+{
+  pname = "sgfutils";
+  version = "unstable-2017-11-27";
```

Comment:
> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
there is no upstream version before the commit, and there are 1526 instances of `unstable-YYYY-MM-DD` versions throughout the repo. what do you suggest?

Reply:
Ok, added 0.25 as the github revision only contains some extra files compared to the tarball, no differences in existing ones.